### PR TITLE
Add DOCTYPE and specify encoding

### PR DESCRIPTION
**Pull request by mathiasbynens**
Since “…” is a non-ASCII character, the encoding of the HTML document should be specified.

### CODE REVIEW

**Comment by mathiasbynens** [comment]
Btw, the HTML could be simplified to:

```html
<!DOCTYPE html>
<meta charset="utf-8">
<title>Redirecting…</title>
<script>
  location = '${process.env.REDIRECT_URL}?${querystring.stringify(data)}';
</script>
```

If you care about non-JS support:

```js
<meta http-equiv="refresh" content="0;URL=${ url }">
```

**Comment by mxstbr** [comment]
> Btw, the HTML could be simplified to:

Let's simplify it to that then, including the non JS support! Thanks for the PR 😊

File: index.js [hunk 1]
```diff
@@ -4,17 +4,13 @@ const querystring = require('querystring')
 const axios = require('axios')
 const { send } = require('micro')
 
+const url = `${process.env.REDIRECT_URL}?${querystring.stringify(data)}`
```
Comment (mxstbr) [inline index.js]: That's not going to work since `data` is undefined in that scope 😉

**Review by mxstbr — CHANGES_REQUESTED** [review]

**Comment by mathiasbynens** [comment]
Done.

Note that the code does not sanitize `process.env.REDIRECT_URL` in any way. My patch didn’t change that (that would have to be a separate patch anyway), but please consider doing so.

**Comment by mxstbr** [comment]
> but please consider doing so.

Hmm, why/how? That's developer submitted, could that be exploited?

File: index.js (hunk 1 above)
Comment (mathiasbynens) [reply index.js]: No kidding 😊  Fixed!

**Comment by mathiasbynens** [comment]
Just pointing it out! As a best practice, anything injected into an HTML document should be escaped based on the context. In this case it only matters if the developer (or someone else) sets a malicious environment value, so it’s not a big deal in this particular scenario. (Still, people might copy-paste this part of the code and use it in other scenarios etc. so it might be worth it to set a good example and sanitize it.)

**Comment by mxstbr** [comment]
Good call, would appreciate a follow-up PR!

I think with the latest changes we're now missing a closing bracket for the function?

**Comment by mathiasbynens** [comment]
Geez, how many typos could I make in a single PR?! Updated

**Comment by mxstbr** [comment]
I'll add CI so this doesn't happen anymore, thanks for going through the troubles!

**Review by mxstbr — APPROVED** [review]

**Comment by mxstbr** [comment]
Again, would appreciate a follow-up PR with the sanitization 😊

**Pull request merged**